project/Project.scala

```diff
@@ -379,9 +383,10 @@ object Zipkin extends Build {
     ).settings(
       libraryDependencies ++= Seq(
         twitterServer,
-        "org.apache.kafka" %% "kafka" % "0.8.1.1",
+        "org.apache.kafka" %% "kafka" % "0.8.2.1",
         "org.apache.commons" % "commons-io" % "1.3.2",
-        scroogeDep("serializer")
+        scroogeDep("serializer"),
+        "com.github.charithe" % "kafka-junit" % "1.3" % "test"
```

Comment:
@abesto everything above the kafka-junit line should be done to the build regardless of my change